### DISCUSSION TITLE
CPU-only build

### DISCUSSION THREAD

**Discussion by n-west** [NVIDIA/MatX#496] [Q&A] [answered]
I'd like to be able to develop without cuda installed and be able to run on systems with or without gpus. Right now the MatX build errors out looking for cuda and nvcc. Is it possible to compile and run in a cpu-only mode?

**Comment by cliffburdick** [comment, marked as answer]
Hi @n-west, right now you need cuda installed even if you're just developing on the host. This is something we may change in the future, but there are a lot of dependencies on cuda right now. You could try compiling inside a cuda container to make it a bit easier.
